### PR TITLE
Updated bounding box functions to use Keras_cv

### PR DESCRIPTION
**Pull request by KelvYing**
Change implementations of bounding box loss function to use functions from Keras_cv library
Updated tests for function and included keras_cvin requirements

### CODE REVIEW

File: src/image_utils.py [hunk 1]
```diff
@@ -5,6 +5,9 @@
 import matplotlib.pyplot as plt
 from ipywidgets import interact
 import math
+from keras_cv import losses
+
+os.environ['TF_ENABLE_ONEDNN_OPTS'] = '0'
```
Comment (pijner) [inline src/image_utils.py]: Any reason for disabling this? It usually boosts performance on CPU a fair bit

Comment (KelvYing) [reply src/image_utils.py]: Oh my bad I forgot to remove that.

**Review by pijner — APPROVED** [review]
